wgpu-types/src/lib.rs

```diff
@@ -128,6 +123,39 @@ impl From<Backend> for Backends {
     }
 }
 
+impl Backends {
+    /// Generates a set of backends from a comma separated list of case-insensitive backend names.
+    ///
+    /// Whitespace is stripped, so both 'gl, dx12' and 'gl,dx12' are valid.
+    //
+    /// Names:
+    /// - vulkan = "vulkan" or "vk"
+    /// - dx12   = "dx12" or "d3d12"
+    /// - dx11   = "dx11" or "d3d11"
+    /// - metal  = "metal" or "mtl"
+    /// - gles   = "opengl" or "gles" or "gl"
+    /// - webgpu = "webgpu"
+    pub fn from_comma_list(string: &str) -> Option<Self> {
```

Comment:
can this live in wgpu-core instead of wgpu-types?

Reply:
Addressed.